src/main/java/mods/eln/Eln.java

```diff
@@ -4799,6 +4821,15 @@ void registerPortableItem(int id) {
 		}
 	}
 
+	private void registerFuelBurnerItem(int id) {
+		sharedItemStackOne.addElement(0 + (id << 6),
+			new FuelBurnerDescriptor(I18N.TR_NAME(Type.NONE, "Small Fuel Burner"), 5000 * fuelHeatFurnacePowerFactor, 2, 1.6f));
```

Comment:
Oh, this factor can be used to balance the fuel heat furnace power in the ELN config file like for a lot of other machines, it has nothing to do with the fuel consumption.